pkgs/applications/misc/khard/default.nix

```
@@ -1,12 +1,14 @@
-{ lib, python3, fetchPypi, khard, testers }:
+{ lib, python3, fetchPypi, khard, testers, installShellFiles }:
 
 python3.pkgs.buildPythonApplication rec {
-  version = "0.18.0";
+  version = "0.19.0";
   pname = "khard";
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
Could you please use a parameter, `pyproject`, instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters